test/plugin/test_exception_detector.rb

```diff
@@ -270,6 +270,222 @@ class ExceptionDetectorTest < Test::Unit::TestCase
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_STACK_ERR = <<END.freeze
+Unhandled exception:
+Stack Overflow
+#0      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:33:20)
+#1      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:61:13)
+#2      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:33:3)
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_FALLTHROUGH_ERR = <<END.freeze
+Unhandled exception:
+'null': Switch case fall-through at line null.
+#0      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:39:20)
+#1      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:51:13)
+#2      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:39:3)
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_TYPE_ERR = <<END.freeze
+Unhandled exception:
+'file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart': malformed type: line 7 pos 24: cannot resolve class 'NoType' from '::'
+  printError( () { new NoType(); } );
+                       ^
+
+
+#0      _TypeError._throwNew (dart:core-patch/errors_patch.dart:82)
+#1      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:7:24)
+#2      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:36:13)
+#3      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:7:3)
+#4      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_FORMAT_A_ERR = <<END.freeze
+Unhandled exception:
+FormatException: format exception
+#0      main.<anonymous closure> (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:25:20)
+#1      printError (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:42:13)
+#2      main (file:///Users/westoncooper/LN/fluent-plugin-detect-exceptions/test/errorTest.dart:25:3)
+#3      _startIsolate.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:265)
+#4      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:151)
+END
+
+  DART_FORMAT_B_ERR = <<END.freeze
```

Comment:
How about calling the above `DART_FORMAT_ERR`, and this one `DART_FORMAT_WITH_CODE_ERR`?